packages/react-jss/src/withStyles.js

```diff
@@ -113,7 +114,15 @@ const withStyles = <Theme>(styles: Styles<Theme>, options?: HOCOptions<Theme> =
         super(props)
 
         this.state = WithStyles.createState(props)
-        WithStyles.manage(props, this.state)
+        if (isInBrowser) {
+          WithStyles.manage(props, this.state)
+        } else {
+          const {registry} = props.jssContext
```

Comment:
This should also run on the client to be in sync with the hook version

Reply:
Or, should we change the hooks version to be like this instead? After all, `manageSheet()` also calls `registry.add()`.

Reply:
For now though I'll do as the hooks API does